transport-classes-io_uring/src/main/java/io/netty/channel/uring/IoUringIoHandler.java

```diff
@@ -428,8 +439,21 @@ public static IoHandlerFactory newFactory() {
      * @return              factory
      */
     public static IoHandlerFactory newFactory(int ringSize) {
+        IoUringIoHandlerConfiguration configuration = new IoUringIoHandlerConfiguration();
+        configuration.setRingSize(ringSize);
+        return () -> new IoUringIoHandler(configuration);
+    }
+
+    /**
+     * Create a new {@link IoHandlerFactory} that can be used to create {@link IoUringIoHandler}s.
+     * Each {@link IoUringIoHandler} will use same option
+     * @param option the io_uring option
```

Comment:
this parameter should be plural; `options`

Reply:
renamed to config.